Mobile/ShopLens/Droid/Assets/App.config

```diff
@@ -1,26 +1,79 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <configuration>
 <appSettings>
+  
+  <!-- Custom Vision -->
   <add key="cvProjectId" value="11e80156-6630-4cd7-a9e6-3441f9da09e2" />
   <add key="cvPredictionKey" value="13185045e5b84af5bd045d029447d951" />
   <add key="cvRequestUri" 
        value="https://southcentralus.api.cognitive.microsoft.com/customvision/v2.0/Prediction/11e80156-6630-4cd7-a9e6-3441f9da09e2/image" />
   <add key="webClassifierImgSize" value ="512" />
+  
+  <!-- Input/Output -->
   <add key="fileProviderName" value=".shoplens.fileprovider" />
+  
+  <!-- Camera activity -->
   <add key="CmdWhatIsThis" value="what is this" />
   <add key="CmdPickPhoto" value="I have a photo" />
-  <add key="CmdVoiceCart" value="voice cart" />
-  <add key="CmdAddCartList" value="add" />
-  <add key="CmdVoiceList" value="voice list" />
-  <add key="ShopCartPrefs" value="USER_SHOP_CART" />
+  
+  <!-- Text To Speech -->
+  <add key="AnswerUtteranceId" value="NEED_USER_ANSWER" />
   <add key="VoicerPauseTime" value="500" />
   <add key="VoicerCheckDelay" value="200" />
+  <add key="InitialTutorialMsg" 
+       value=
+       "Welcome to ShopLens. I am your virtual shopping assistant Jenna, but you can call me Jen.
```

Comment:
I don't know whether these long messages should be in our app.config. I at first thought they should be in some sort of file, maybe JSON, I just put them here for now.

Reply:
LARGE